Source/Core/DolphinQt/Config/ControllerInterface/ServerStringValidator.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 Dolphin Emulator Pro
```

Comment:
Is this the new Patreon supporters version? :-D

Reply:
LOL.  I copied that from some other file... I'll fix it, thanks!